src/js/controls/Tab.js

```diff
@@ -193,17 +198,26 @@ lm.utils.copy( lm.controls.Tab.prototype, {
 	 */
 	_onTabClick: function( event ) {
 		// left mouse button or tap
-    	if( event.button === 0 ) {
+    	if( !event || event.button === 0 ) {
```

Comment:
When could this event be null?

Reply:
In header.js on key.Enter I re-used the _onTabClick function to set the active tab... a little messy.

`this.tabs[this.dropdownKeyIndex]._onTabClick();`